playbooks/ci-full/templates/heat_stack.yml

```diff
@@ -0,0 +1,254 @@
+heat_template_version: 2013-05-23
+
+description: HOT template to deploy Ursula Tests env
+
+# Note: there should be a placeholder line after each "if-else" block to keep the right format of the template
+
+parameters:
+  image:
+    type: string
+    description: Name of image to use for servers
+    {% if ansible_env.IMAGE_ID -%}
+    default: {{ ansible_env.IMAGE_ID }}
+    {% else -%}
+    default: ubuntu-14.04
+    {% endif -%}
```

Comment:
This approach doesn't look right to me. I'm not sure why we'd need to change the default, when these parameters allow us to specify an image when we use heat and override the default value of ubuntu-14.04